src/style.css

```diff
@@ -0,0 +1,10 @@
+@import url("./styles/cards.css");
+
+body {
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
if you want to apply ` box-sizing: border-box;` to each element, move it to `*` selector